### DISCUSSION TITLE
compiling error ?

### DISCUSSION THREAD

**Discussion by RootnuII** [YimMenu/YimMenu#1445] [Q&A] [answered]
hi im getting an error i git cloned it im in visual studio and im getting this odd error
Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	E2500	a literal operator template must have a template parameter list equivalent to '<char ...>'	YimMenu.dll - x64-Release	C:\Users\nullsec\Desktop\YimMenu-master\YimMenu\src\services\translation_service\translation_service.hpp	88	
![image](https://github.com/YimMenu/YimMenu/assets/66104474/a7088d6c-673e-4f4b-a99b-edd54f082328)

**Comment by Yimura** [comment, marked as answer]
Are you using the latest version of Visual Studio? VS 2019 does not work to compile YimMenu, you need VS 2022.

**Reply by RootnuII** [reply]
thanks that seemed to work but then it created these lol 
![image](https://github.com/YimMenu/YimMenu/assets/66104474/6bda68f0-1e11-4655-93d0-ccdf66a10ac5)

**Reply by RootnuII** [reply]
it took out 	constexpr auto operator"" _T() for some reason

**Reply by RootnuII** [reply]
got everything to work
